app/models/user.py

```diff
@@ -23,9 +25,15 @@ class Role(db.Model):
     @staticmethod
     def insert_roles():
         roles = {
-            'User': (
+            'Inactive': (
```

Comment:
Might as well delete that role in the same commit that nukes the whole concept of a "default role." I agree both should be done, but they should both be done together with specific intent. i.e. let's figure out how admin registration will work, completely remove the ability to make a user that'd have the default role, and then get rid of the default role.